earth_enterprise/rpms/opengee-fusion/snippets/post-install.sh

```diff
@@ -150,10 +150,10 @@ install_or_upgrade_asset_root()
         if [ "$IS_SLAVE" = "false" ]; then
             OWNERSHIP=`find "$ASSET_ROOT" -maxdepth 0 -printf "%g:%u"`
             if [ "$OWNERSHIP" != "$GEGROUP:$GEFUSIONUSER" ] ; then
-                NOCHOWN=""
+                DOCHOWN="--chown"
```

Comment:
Just a thought: perhaps in this case we should just print a message and not do the chown. I'm not sure which would be a better user experience - a possible surprise that the install takes a long time or a possible surprise that the permissions are not right. I'm good either way.

Reply:
Yeah, best to be consistent.  Since we decided not to chown the publish root, I'll remove the chown here as well.